Trimble.Modus.Components/Controls/TMModal/TMModalContents.xaml.cs

```diff
@@ -120,6 +158,10 @@ internal TMModalContents(string titleText, string messageText = null, ImageSourc
         FullWidthButton = fullWidthButton;
         Message = messageText;
         BindingContext = this;
+
+        // Bind color to dynamic resource
+        this.SetDynamicResource(TextAndImageColorProperty, "ModalTextAndImageColor");
+        this.SetDynamicResource(ModalBackgroundColorProperty, "ModalBackgroundColor");

```

Comment:
Can you use the 'theme.xaml' like in the other controls? It makes the job easier when you to do this for multiple properties